.pre-commit-config.yaml

```diff
@@ -33,3 +33,9 @@ repos:
       name: format_checkers
       entry: python cve_bin_tool/format_checkers.py
       files: "^cve_bin_tool/checkers/__init__.py"
+
+-   repo: https://github.com/compilerla/conventional-pre-commit
+    rev: 16bae86cd914c7c9ddc695d54e50052a69e8b05c
```

Comment:
Is there a reason to use a git sha rather than the tag they provide?

Reply:
There is an open issue with `pre-commit autoupdate` downgrading the versions of the pre-commit hook. Once that's fixed we can replace the sha with the tags. Refer https://github.com/compilerla/conventional-pre-commit/issues/10

Reply:
Just ran the command with sha in `rev` and the bug made it revert to v1. Seems like we are better off using tags anyway (Thought sha would be equivalent of pinning the version, my bad).